### PR TITLE
hw14

### PR DESCRIPTION
**Pull request by breengles**

### CODE REVIEW

File: models/hw14.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+        emb = self(imgs_q)
+
+        dist = self.__pairwise_distance(emb, protos)
+
+        log_p = F.log_softmax(-dist, dim=-1)
+        log_p = log_p[torch.arange(log_p.shape[0]), uniq_labels_q_idx]
+
+        J = 0
+        for k in uniq_labels_s:
```
Comment (kraalfar) [inline models/hw14.py:79]: чтобы цикла не было можно было labels как столбик приравнять к uniq_labeles_s, получить матрицу, где в строчке по одному true и от нее аргмакс взять. а потом уже можно обычную кросс энтропию считать
